tests/examples/policy_rule_c.json

```diff
@@ -3,7 +3,7 @@
     "rules" : [
         {
           "AND": [{
-            "EQUAL" : {"user_name" : "admin" }
+            "EQUAL" : {"userName" : "admin" }
```

Comment:
Same as the other case: is this change from "user_name" breaking the policy check?